doc/Getting_Started.md

```diff
@@ -11,8 +11,29 @@ developing Tock.
 3. [arm-none-eabi toolchain](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads) (version >= 5.2)
 4. Command line utilities: wget, sed, make, cmake
 
+### Super Quick Setup
+
+MacOS:
+```
+$ curl https://sh.rustup.rs -sSf | sh
+$ brew tap ARMmbed/homebrew-formulae && brew update && brew install arm-none-eabi-gcc
+$ pip3 install tockloader --user
```

Comment:
Do we want to put `--user` here? You need to do extra things to get user pip working out of the box, and brew's installation of pip is a no-sudo, no-user IIRC

Reply:
I agree better with ```sudo``` but I can only speak for ```Linux/ubuntu``` where you must be a member of the ```dialout group``` in order to access to serial port without being ```root```

Reply:
I just matched what the tockloader docs say. I'm old school and have never tried --user.

Reply:
Ah, it was breaking it out into OS X and Linux separately here that called more attention to it. I'll push a fix for that now and for @niklasad1's dialout point